docs/changes/2234.feature.rst

```diff
@@ -0,0 +1,5 @@
+``QualityQuery`` is now a ``TelescopeComponent``.
+
+You can now give different quality queries per telescope type or id.
+Change is backwards compatible, older configurations still work and
+set the query-expression to the global value of the TelescopeComponent.
```

Comment:
But as of now the API of QualityQuery is changed in an incompatible way. If you include the `key=None`, it will be truly backwards compatible.